recipes/cairo/all/conanfile.py

```diff
@@ -252,36 +266,42 @@ def package_info(self):
                 self.cpp_info.components["cairo_"].requires.extend(["xorg::xcb-render"])
             if self.options.with_xlib:
                 self.cpp_info.components["cairo_"].requires.extend(["xorg::x11", "xorg::xext"])
+
         if tools.is_apple_os(self.settings.os):
             self.cpp_info.components["cairo_"].frameworks.append("CoreGraphics")
 
-
         if self.settings.os == "Windows":
-            self.cpp_info.components["cairo-win32"].names["pkg_config"] = "cairo-win32"
+            self.cpp_info.components["cairo-win32"].set_property("pkg_config_name", "cairo-win32")
```

Comment:
Please leave the old names for v1 generators

Reply:
Will the default name not be the name of the component anyway?

Reply:
We like to be explicit 😄 however I am sure this suggestion will change in the near future